examples/repl.rs

```diff
@@ -63,98 +65,97 @@ fn prepare_home_dir() -> PathBuf {
 fn main() {
     env_logger::init();
 
-    let app = cli::make_cli_subcommands();
-    let mut repl_app = app.clone().setting(AppSettings::NoBinaryName);
-
-    let app = cli::add_global_flags(app);
-
-    let matches = app.get_matches();
+    let cli_opt: WalletOpt = WalletOpt::from_args();
 
     // TODO
-    // let level = match matches.occurrences_of("v") {
-    //     0 => LevelFilter::Info,
-    //     1 => LevelFilter::Debug,
+    // let level = match cli_opt.verbosity.map(|v| v.len()) {
+    //     Some(0) => LevelFilter::Info,
+    //     Some(1) => LevelFilter::Debug,
     //     _ => LevelFilter::Trace,
     // };
 
-    let network = match matches.value_of("network") {
-        Some("regtest") => Network::Regtest,
-        Some("testnet") | _ => Network::Testnet,
-    };
+    let network = Network::from_str(cli_opt.network.as_str()).unwrap_or(Network::Testnet);
```

Comment:
The `match` we were doing before here was mostly to disallow using mainnet for safety reason.. Maybe it still makes sense to do it?

Or ideally (that would be better than what was here before) we could either exit with an error or maybe print a warning that this is a development tool and it should be used carefully on mainnet

Reply:
I'm going to just a add a warning message if `bitcoin` is selected. There are certainly safe uses for `bdk` on mainnet such as using  monitoring / watchonly testing.

Reply:
Agreed, the quality has definitely improved since I originally wrote that code. A warning is probably more suitable given the current state of our code

Reply:
great, added a warning.